pkg/pb/inspector.proto

```diff
@@ -198,3 +206,33 @@ message DashboardResponse {
   google.protobuf.Timestamp last_pinged = 8;
   google.protobuf.Timestamp last_queried = 9;
 }
+
+message SegmentHealthRequest {
+  bytes bucket = 1;
+  bytes encrypted_path = 2;
+  int64 segment = 3;
```

Comment:
field 'segment' felt bit confusing. Either renaming 'index' would be inline with response or put a comment saying field segment refers to segment's index.

Reply:
renamed